tools/tcpcong_example.txt

```diff
@@ -0,0 +1,463 @@
+192.168.219.3/39070   192.168.219.1/19225   668     4      32      455     0      1706
+192.168.219.3/39098   192.168.219.1/19225   692     4      38      424     0      2110
+192.168.219.3/39112   192.168.219.1/19225   564     0      2       593     0      2291
+192.168.219.3/39120   192.168.219.1/19225   599     0      4       555     0      2387
+
+07:39:12
+LAddrPort            RAddrPort             Open_ms Dod_ms Rcov_ms Cwr_ms  Los_ms Chgs 
+192.168.219.3/39070   192.168.219.1/19225   576     3      27      391     0      1525
+192.168.219.3/39098   192.168.219.1/19225   580     3      36      379     0      1893
+192.168.219.3/39112   192.168.219.1/19225   474     1      10      512     0      2009
+192.168.219.3/39120   192.168.219.1/19225   505     1      9       483     0      2022
+
+07:39:13
+LAddrPort            RAddrPort             Open_ms Dod_ms Rcov_ms Cwr_ms  Los_ms Chgs 
+192.168.219.3/39070   192.168.219.1/19225   546     6      27      418     0      1659
+192.168.219.3/39098   192.168.219.1/19225   564     4      40      390     0      1937
+192.168.219.3/39112   192.168.219.1/19225   479     0      3       514     0      2008
+192.168.219.3/39120   192.168.219.1/19225   515     0      4       479     0      1982
+
+The (-u) option can be specified for recording the duration as miroseconds.
+Eg printing output every 1 second, and including timestamps (-T) and 
+microseconds (-u) for local ports 30000-40000 and remote ports 19225-19227:
+./tcpcong -T -u -L 30000-40000 -R 19225-19227 1 3 
+Tracing tcp congestion control status duration... Hit Ctrl-C to end.
+
+07:39:44
+LAddrPort            RAddrPort             Open_us Dod_us Rcov_us Cwr_us  Los_us Chgs 
+192.168.219.3/39070   192.168.219.1/19225   600971  3232   38601   509796  0      1843
+192.168.219.3/39098   192.168.219.1/19225   667184  5585   26285   453575  0      1969
+192.168.219.3/39112   192.168.219.1/19225   580982  22     1502    569479  0      2210
+192.168.219.3/39120   192.168.219.1/19225   600280  201    955     550752  0      2327
+
+07:39:45
+LAddrPort            RAddrPort             Open_us Dod_us Rcov_us Cwr_us  Los_us Chgs 
+192.168.219.3/39070   192.168.219.1/19225   567189  2029   25966   404698  0      1612
+192.168.219.3/39098   192.168.219.1/19225   597201  2263   24073   376454  0      1578
+192.168.219.3/39112   192.168.219.1/19225   500792  846    9297    489264  0      1850
+192.168.219.3/39120   192.168.219.1/19225   518700  94     749     480171  0      1967
+
+07:39:46
+LAddrPort            RAddrPort             Open_us Dod_us Rcov_us Cwr_us  Los_us Chgs 
+192.168.219.3/39070   192.168.219.1/19225   587340  5324   37035   370066  0      1602
+192.168.219.3/39098   192.168.219.1/19225   532986  5630   31624   345336  0      1319
+192.168.219.3/39112   192.168.219.1/19225   481936  1129   6244    510235  0      1909
+192.168.219.3/39120   192.168.219.1/19225   507196  316    6200    485737  0      1957
+
```

Comment:
Could you have at least one ipv6 examples so we could see what it looks like?

Reply:
okay, i will add it in example file.